pitch.md

```diff
@@ -1,20 +1,86 @@
 
 ## Describe your project in five words
+Describe schedules sucinctly and precisely.
 
 ## Audience
 _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
 them?_
 
+This project is designed to serve individuals who need to schedule meetings
+among a group of people.  The language would provide a means by which an
+individual can specify their preference related to scheduling.  With more than
+a handful of reasonably busy individuals it is unlikely that there will be a
+meeting time which is the top choice of everyone involved.
+
+Currently the de-facto standards for scheduling are one
+of the following two methods:
+ * Everyone send out their availability for a small set of times and a leader
+picks the times that seem to work best for a plurality.
+ * Everyone uses some software, such as a WhenIsGood&#8482; or When2Meet&#8482;
+and choses a binary yes/no for each time slot.
+
+Beyond the issue of there not being a single time that works for everyone,
+there is also an issue in which individuals will be conservative in listing
+times that they could theoretically attend, as Daniel Sonner pointed out,
+some people will paint over the times that are *best* for them, but not
+*all* the times which could work for them.
+
+Both of these issues (inability to find a time preferred by all individuals
+and people being extra conservative with respect to their availability) are
+addressed by allowing for greater expressiveness in describing one's schedule.
+This is because allowing for a gradation of availability (from totally free, to
+somewhat available, to completely busy) might make people less likely to
+game the system by saying that they are completely at times that are just
+suboptimal for them.  It also means that there is more information which a
+clever algorithm could make of when deciding which of two conflicts is worse.
+
+If I was able to help them, users would be able to specify, in a way that feels
+natural, their availability and preferences.  Hopefully this can reduce 
+the work of an individual (who must compile the results of an email) or the
+repetition inherent in painting over the same boxes for pairs of weeks when
+using a WhenIsGood&#8482;.
+
 
 ## Why a language?
 _Why is a **DSL** appropriate for your user(s)? How does it address the need?_
 
+The essential gap I am trying to bridge is that of language.  I hope to provide
+users with a better way of expressing their schedules, to satisfy two specific
+goals:
+ * Less redundancy: No need to paint the same pattern over multiple weeks.
+ * More detailed information: Can specify a range of availability rather than 
+ a binary variable for each time slot.
+
+A simple calendar or WhenIsGood does not allow for this level of
+expressiveness. An user actually writing text (such as in the email scenario)
+does allow for arbitrarily high expressivenes, but would then shift the burden
```

Comment:
I think the point about the over expressiveness and flexibility allowed by email for scheduling is worth adding more anecdotal points about. I agree that it is definitely an unideal situation when their is an email thread of a group of people trying to figure out a suitable time.

I would keep this in mind as you decide what features to include in your languaage.